libraries/TeamSpeak3/Node/Server.php

```diff
@@ -89,6 +89,28 @@ public function request($cmd, $throw = TRUE)
 
     return $this->getParent()->request($cmd, $throw);
   }
+  
+  /**
+   * Returns an array filled with TeamSpeak3_Node_Server objects.
+   *
+   * @param  array $filter
+   * @return array
+   */
+  public function serverInfo()
+  {
+	return $this->request("serverinfo")->toList();
```

Comment:
Looks like this is handled by: `TeamSpeak3_Node_Abstract->getInfo()`.
Ref: https://docs.planetteamspeak.com/ts3/php/framework/class_team_speak3___node___abstract.html#ac60d6af1acba9d78d719d4b86e987c07

Example to get server properties for 'virtual server' with id `1`:
```php
require_once 'TeamSpeak3.php';

$ts3_host = TeamSpeak3::factory("serverquery://username:password@255.255.255.255:10011");
$ts3_virtual_server = $ts3_host->serverGetById(1); // Where `1` is desired virtual server ID.

print_r($ts3_virtual_server->getInfo(true, true)); // Arguments specify extended, convert to human readable.
```